src/auth/user.ts

```diff
@@ -34,3 +34,32 @@ export interface User extends ArenaObject<UserProps> {
   prefs?: UserPrefs
   status: UserStatus
 }
+
+
+// Methods
+
+const getAuthGroups = (user:User):AuthGroup[] => user?.authGroups || []
+
+const isSystemAdmin =  (user:User):boolean => (getAuthGroups(user) || []).some(AuthGroupMethods.isSystemAdminGroup)
+
+const getAuthGroupBySurveyUuid= (surveyUuid: string | undefined, includeSystemAdmin = true) => (user: User):AuthGroup | undefined => {
```

Comment:
never do this: surveyUuid: string | undefined
always use conditional parameter surveyUuid?:string